packages/react-components/react-infobutton/package.json

```
@@ -32,6 +32,7 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@fluentui/react-label": "^9.1.2",
     "@fluentui/react-icons": "^2.0.175",
```

_nit:_ Keep alphabetical order

```suggestion
    "@fluentui/react-icons": "^2.0.175",
    "@fluentui/react-label": "^9.1.2",
```